src/project/internal/projectactionscontroller.cpp

```
@@ -84,6 +84,39 @@ void ProjectActionsController::init()
+    }
+
+    prj->trackRemoved().onReceive(this, [this](const trackedit::Track&) {
+        m_actionEnabledChanged.send({ "file-share-audio" });
+    }, muse::async::Asyncable::Mode::SetReplace);
+
+    prj->trackClipListChanged().onReceive(this, [this](const trackedit::Track&) {
+        m_actionEnabledChanged.send({ "file-share-audio" });
+    }, muse::async::Asyncable::Mode::SetReplace);
+
+    m_actionEnabledChanged.send({ "file-share-audio" });
+}
```

Cppcheck (2.20.0)

[style] The function 'isStandardTag' is never used.

(unusedFunction)